### ISSUE TITLE
TabControl notifies screen readers of its role on every tab change.

### ISSUE THREAD

**Issue by trypsynth** [dotnet/winforms#13004]
### .NET version

9.0.200

### Did it work in .NET Framework?

Not tested/verified

### Issue description

When creating a TabControl and arrowing left/right between tabs with a screen reader, you hear the text "tab control" before the name of the tab every time. This is inconsistent with standard Win32 behavior (i.e. the role is announced on focus, but only the tab name is announced when you switch tabs).

### Steps to reproduce

1. Run the following code with NVDA or Narator:
```
namespace TabRoleTest;

public class Form1 : Form {
	private TabControl tabs;
	private TabPage firstTab, secondTab;

	public Form1() {
		Text = "Tab Control Test";
		ClientSize = new Size(640, 480);
		tabs = new TabControl {
			Dock = DockStyle.Top,
			Height = 250
		};
		firstTab = new TabPage("First tab");
		secondTab = new TabPage("Second tab");
		tabs.TabPages.AddRange(new[] {firstTab, secondTab});
		Controls.Add(tabs);
	}

	[STAThread]
	static void Main() {
		ApplicationConfiguration.Initialize();
		Application.Run(new Form1());
	}
}
```
2. Left/right arrow through the tabs, and observe what you hear.

**Comment by MelonWang1** [comment]
This issue also repro in .NET6.0, cannot repro in .NET Framework project. Not repro in Narrator and JAWS tool.
NVDA:

![Image](https://github.com/user-attachments/assets/58994645-9f8f-48f3-a7e9-f13f4a1b3e1d)

![Image](https://github.com/user-attachments/assets/024d2f08-396a-41d2-8e43-82d8cf04e870)

Narrator: In framework project, it read 2 times.

![Image](https://github.com/user-attachments/assets/c334827c-cace-4930-ada1-a708c3e49c6f)

![Image](https://github.com/user-attachments/assets/c3b85fef-eec0-4368-8b24-717be16d7aba)

**Comment by Epica3055** [comment]
When we switch tabs WmSelChange will be triggered.

https://github.com/dotnet/winforms/blob/8985769b8a188df3b1d68adbbf9ac81df2bee83d/src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabControl.cs#L1948-L1978

code in line 1965 notify the client to change focus.
```
SelectedTab.TabAccessibilityObject.RaiseAutomationEvent(UIA_EVENT_ID.UIA_AutomationFocusChangedEventId); 
```

![Image](https://github.com/user-attachments/assets/b04e56ff-7b22-4b20-bc01-44a36f38aad9)


As you may see that sometimes TabControl get focused.

I think that's because Native TabControl send the FocusChanged acc Event (since I can't find any related code in our winform repo, and we can clearly see that the event is emitted through AccEvent tool)



![Image](https://github.com/user-attachments/assets/04ca4207-4dbc-401a-9734-9465fb9e9c4a)

I think that's the reason I we can see so many extra 'tab control's.

![Image](https://github.com/user-attachments/assets/546de3ac-d19d-490c-930e-e85eb51813c1)

The reason native TabControl send the acc event I think is the code here
https://github.com/dotnet/winforms/blob/8985769b8a188df3b1d68adbbf9ac81df2bee83d/src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabControl.TabControlAccessibleObject.cs#L128-L137

if we comment out this code
```
        internal override VARIANT GetPropertyValue(UIA_PROPERTY_ID propertyID)
            => propertyID switch
            {
                //UIA_PROPERTY_ID.UIA_HasKeyboardFocusPropertyId => (VARIANT)(this.TryGetOwnerAs(out TabControl? owner) && owner.Focused),
                //UIA_PROPERTY_ID.UIA_IsKeyboardFocusablePropertyId
                //    // This is necessary for compatibility with MSAA proxy:
                //    // IsKeyboardFocusable = true regardless the control is enabled/disabled.
                //    => VARIANT.True,
                _ => base.GetPropertyValue(propertyID)
            };


```
We can see through AccEvent Tool that TabControl don't send acc Focus Event any more
And accordingly, NVDA will work fine.
![Image](https://github.com/user-attachments/assets/662adcfb-4fde-47bb-9e2c-91635e783220)

The downside is that when the Form is opened and TabControl is Focused , TabControl won't be focused by accessible tool automatically.

**Comment by LeafShi1** [comment]
This issue only occurs in NVDA, and is just additional information rather than a violation.
Currently we have other higher priority issues to deal with, so we're closing this issue, and we'll revisit it if more users complain about it.
